apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,31 @@
+#include "TutorialServo.h"
+
+#include <cstdlib>
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees = 180.0, float minPulsewidthInMs = 1,
+                             float maxPulsewidthInMs = 2) {
```

Comment:
The default values should only be stated in the function _declaration_ (your header file). Remove them from the function _definition_